raiden_contracts/deploy/__main__.py

```diff
@@ -377,5 +452,152 @@ def register_token_network(
     )
 
 
+def store_deployment_info(deployment_info: dict, deployment_file_path: Path):
+    with deployment_file_path.open(mode='w') as target_file:
+        target_file.write(json.dumps(deployment_info))
+
+    print(
+        f'Deployment information for chain id = {deployment_info["chain_id"]} '
+        f' has been updated at {deployment_file_path}.',
+    )
+
+
+def verify_deployed_contracts(web3, contract_manager, deployment_file_path: Path):
+    try:
+        with deployment_file_path.open() as deployment_file:
+            deployment_data = json.load(deployment_file)
+    except (JSONDecodeError, UnicodeDecodeError) as ex:
+        raise DeploymentVerificationError(f'Cannot load deployment data file: {ex}') from ex
```

Comment:
should this be `{str(ex)}`?

Reply:
@LefterisJP `{ex}` == `{ex!s}` == `{str(ex)}`, afaict

Reply:
@andrevmatos Is this an f-string thing? Or do we always wrap exceptions in `str()` in the client due to the struct logger? Cause in the client we do a lot of:

```
                except RaidenRecoverableError as e:
                    log.error(str(e))
```

If we don't wrap it in `str()` there, then we get the `TypeError: unsupported operand type(s) for +=: 'RaidenRecoverableError' and 'str'`

But perhaps seems like it is a structlogger thing.

Reply:
Not sure about `structlogger`, but: https://www.python.org/dev/peps/pep-0498/#s-r-and-a-are-redundant, also supports lambdas etc.